package.json

```diff
@@ -27,19 +27,595 @@
   },
   "homepage": "https://github.com/mladoniczky/grovejs#readme",
   "devDependencies": {
-    "clean-webpack-plugin": "^2.0.0",
-    "documentation": "^9.3.1",
-    "jasmine-core": "^3.3.0",
-    "karma": "^4.0.1",
+    "@types/jest": "^29.5.14",
+    "clean-webpack-plugin": "^2.0.2",
+    "documentation": "^14.0.3",
+    "jasmine-core": "^3.99.1",
+    "karma": "^6.4.4",
     "karma-chrome-launcher": "^2.2.0",
     "karma-coverage": "^1.1.2",
     "karma-jasmine": "^2.0.1",
     "karma-jasmine-html-reporter": "^1.4.0",
     "karma-mocha-reporter": "^2.2.5",
-    "karma-webpack": "^3.0.5",
-    "webpack": "^4.29.6",
-    "webpack-cli": "^3.2.3",
+    "karma-webpack": "^5.0.1",
+    "ts-loader": "^9.5.1",
+    "typescript": "^5.6.3",
+    "webpack": "^5.96.1",
+    "webpack-cli": "^5.1.4",
     "webpack-dev-server": "^3.2.1"
   },
-  "dependencies": {}
+  "directories": {
+    "test": "test"
+  },
+  "dependencies": {
```

Comment:
Why there are so many dependencies when the library actually uses nothing from this? :thinking: 
The library should be as clean as possible, so most of the dependencies should be dev dependencies.